src/node/psbt.h

```diff
@@ -41,6 +41,14 @@ struct PSBTAnalysis {
         next = PSBTRole::CREATOR;
         error = err_msg;
     }
+
+    void Reset() {
```

Comment:
e769f25031a63c378f2efeb957b90a72a8524a5e

nit, `{` in new line?

Reply:
Removed Reset